package.json

```
@@ -5,7 +5,7 @@
   "description": "Helpful in-browser tooling for Magento 2",
   "license": "SEE LICENSE IN LICENSE",
   "scripts": {
-    "test": "npm run format:check && npm run license:check && jest",
+    "test": "npm run format:check && npm run license:check && tsc && jest",
```

You already have the `ts-jest` preset enabled for Jest, so I'm assuming this `tsc` command is to catch compiler errors and not to actually emit any code? If so, might I suggest:
```suggestion
    "test": "npm run format:check && npm run license:check && tsc --noEmit && jest",
```
Also, probably best not to do this on every test run and to separate out the "check" scripts.

---

> If so, might I suggest

`noEmit` is [actually turned on already](https://github.com/magento/m2-devtools/blob/c2f32bd5e3ecb83adf1bd288f3bc248608f0e901/tsconfig.json#L3), since the project is compiled with Babel.

> probably best not to do this on every test run

`npm test` here is strictly for CI or doing a "full sweep" before pushing code.  Running unit/integration tests locally can be done with `npm run test:dev` or `npx jest --watch`